templates/index.html

```diff
@@ -301,9 +301,9 @@ <h2>Add visibility, auditing and compliance controls to your Juju deployment</h2
           </div>
           <div class="row u-hide" id="search-and-filter">
             {{ image(
-                url="https://assets.ubuntu.com/v1/0a0fc6ba-jaas_landing_page_search_and_filter_final.png",
+                url="https://assets.ubuntu.com/v1/f3ff0a1b-searchandfilter.jpg",
```

Comment:
Just a small subjective comment that you can disregard: the `searchandfilter.jpg` looks slightly out of place next to all the rest of the screenshots, as it is the only one that only shows a cropped piece of the full app.

Reply:
Yep I know what you mean. This is how the previous version had it. I think it's like this because otherwise the search and filter is very small so hard to see the details.